index.html

```diff
@@ -183,35 +183,31 @@ <h3 id="unacceptablebehaviour">
+  <li><strong>Appreciate and Accommodate Our Similarities and Differences</strong>We come from many cultures and backgrounds, ways of life, and standard of behavior. Cultural differences can encompass everything from official religious observances to personal habits to clothing. Be respectful of people with different practices, attitudes, and beliefs. To help us achieve and maintain these high standards, each individual participant is expected to share responsibility for our work environment by adhering to the following behavioral guidelines:
+</li>
 	
-<p>To help us achieve and maintain these high standards, each individual participant is expected to share
-	responsibility for our work environment by adhering to the following behavioral guidelines:</p>
 	
-	<ol>
-		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
+	<ul>
+		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions and beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
 		
-  <li>Never <a>harass</a> or <a>bully</a> anyone verbally, physically or 
-   <a data-lt="sexual harassment">sexually</a>.</li>
-  <li>Never <a>discriminate</a> on the basis of personal characteristics or 
-    group membership.</li>
-  <li>Communicate constructively and avoid <a>demeaning</a> or <a>insulting</a>
-    behavior or language.</li>
-  <li>Seek, accept, and offer objective work criticism, and <a>acknowledge</a> 
-    properly the contributions of others. </li>
-  <li>Be honest about your own qualifications, and about any circumstances that
-    might lead to <a href="/Consortium/Process/policies.html#coi">conflicts of
-    interest</a>.</li>
-	
-  <li>Promote the rules of this Code and take action (especially if you are in
-    a <a>leadership position</a>) to bring the discussion back to a more civil
-    level whenever inappropriate behaviors are observed.</li>
```

Comment:
This sentence is removed and not replaced, and there doesn't seem to be any motivation for its removal.

There is a related statement in the unacceptable behaviours section about non-retaliation, but that's not an equivalent.

I haven't seen any documented consensus to remove this; my own view is that promotion of the code is something that belongs outside the code itself and removal of this sentence is okay.

Reply:
editorial decision to remove it. Thanks for your support.